pandas/tests/io/parser/dtypes.py

```diff
@@ -114,6 +114,17 @@ def test_categorical_dtype(self):
         actual = self.read_csv(StringIO(data), dtype='category')
         tm.assert_frame_equal(actual, expected)
 
+    @pytest.mark.slow
```

Comment:
how slow is this test?

Reply:
On my machine it is about 4.5 seconds for the high memory parser, and 6.5 seconds for low memory and python parsers.

Reply:
The minimal `range` necessary to reproduce the error is `range(524289)`, at least locally for me.  Might be beneficial to lower the `range` in the test to offset some of the slowness?  If so, not sure if we should push it down to the limit, or just go down to something like 600k to leave a little buffer room.

Reply:
Checked and the limit is the same on both ubuntu and OSX so decided to do it as just the limit. This cut down the times to 2.25 seconds and 3 seconds so I removed the slow mark.

Reply:
Added slow mark back. rebased. waiting for tests.